FabricExample/package.json

```diff
@@ -10,30 +10,34 @@
     "postinstall": "patch-package"
   },
   "dependencies": {
-    "@gorhom/bottom-sheet": "^4.6.4",
-    "@react-native-community/blur": "^4.4.0",
+    "@gorhom/bottom-sheet": "4.6.4",
+    "@react-native-community/blur": "^4.4.1",
```

Comment:
Not sure, but somewhere you removed the ^ from the library version, would it be better to follow this approach for the whole file in the future?

Reply:
v5 broke e2e tests 🙂